tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "3.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: verify-repo-signature
+      image: registry.redhat.io/rhtas/cosign-rhel9@sha256:75791f7ee9257e837d1e3db13415d858c397e803ffa4534c51ca19a2ba34b2ab
```

Comment:
Could you use the release-service-utils image? It has cosign in it. Or is there a reason this other image is used?

Reply:
I've tried to use the latest tag of release-service-utils in recent local testing, but it failed to do verification of my testing oci-artifact. Seems it is a cosign version issue. The cosign in release-service-utils is using this version:
```
GitVersion:    v2.1.1
GitCommit:     baf97ccb4926ed09c8f204b537dc0ee77b60d043
GitTreeState:  clean
BuildDate:     2023-06-27T06:57:11Z
GoVersion:     go1.20.5
Compiler:      gc
Platform:      linux/amd64
```

And the one in rhtas/cosign-rhel9  is using this one:
```
GitVersion:    54ada3ad
GitCommit:     
GitTreeState:  
BuildDate:     
GoVersion:     go1.21.3
Compiler:      gc
Platform:      linux/amd64
```
Maybe we should upgrade the cosign in release-service-utils to more recent version?

Reply:
Yes, I would suggest to update the utils image to the newer cosign version. Hopefully then it will work for you.

Reply:
Should I open a jira issue for this?

Reply:
You can just submit a PR to bump the version and explain that it's related to this PR and your jira in the description.

Reply:
https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile#L4 should be the only line you have to change in the PR

Reply:
The tag for your new release-service-utils image is `4576e1e2a30439129cb69c8a4f39f7ced2d44376`

Reply:
Updated the image to this one and shrink the first two steps into one with this image.
Thanks for the info!